public/docs/_examples/animations/ts/app/hero.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@angular/core';
+
+export class Hero {
+  constructor(public name:string,
+              public state = 'inactive') {
+  }
+
+  toggleActivation() {
```

Comment:
In the ^^ animations, I thought the `toggleActivation()` was exclusively so only a single items is active (hence the name **activation**). But you are not notifying the heroList to deselect any other active items.

So I wonder if *activation* is the best wording here.